src/mlpack/methods/ne/population.hpp

```diff
@@ -0,0 +1,89 @@
+/**
+ * @file population.hpp
+ * @author Bang Liu
+ *
+ * Definition of Population class.
+ */
+#ifndef MLPACK_METHODS_NE_POPULATION_HPP
+#define MLPACK_METHODS_NE_POPULATION_HPP
+
+#include <cstddef>
+
```

Comment:
I checked the core.hpp and prereqs.hpp in mlpack, it seems doesn't include <cstddef>.